pkg/mcp/server/list_secrets.go

```diff
@@ -0,0 +1,68 @@
+package server
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mark3labs/mcp-go/mcp"
+
+	"github.com/stacklok/toolhive/pkg/secrets"
+)
+
+// SecretInfo represents secret information returned by list
+type SecretInfo struct {
+	Key         string `json:"key"`
+	Description string `json:"description,omitempty"`
```

Comment:
how is description passed to the secret? i do not see any example of setting up that description